docs/cirq_interface.md

```diff
@@ -76,11 +76,11 @@ This circuit can then be simulated using either `QSimSimulator` or
 acquiring the complete state of a reasonably-sized circuit (~25 qubits on an
 average PC, or up to 40 qubits on high-performance VMs).
 Options for the simulator, including number of threads and verbosity, can be
-set with the `qsim_options` field using the `qsim_base` flag format defined in
-the [usage docs](./usage.md).
+set with the `qsim_options` field, which accepts a `QSimOptions` object as
+defined in [qsim_simulator.py](https://github.com/quantumlib/qsim/blob/master/qsimcirq/qsim_simulator.py).
 
 ```
-qsim_options = {'t': 8, 'v': 0}
+qsim_options = qsimcirq.QSimOptions(cpu_threads=8, verbosity=0)
```

Comment:
The old format is still available, right?

Reply:
Added some clarification and an equivalent dict format to the code block.